config/services.yaml

```diff
@@ -60,6 +60,10 @@ services:
             $nbUsers: '%env(int:APP_NB_USERS)%'
             $nbAvailabilities: '%env(int:APP_NB_AVAILABILITIES)%'
 
+    App\DataFixtures\Faker\Provider\:
```

Comment:
As fixtures bundles are only loaded in test environment, we should ignore the `App\DataFixtures` directory from this file (cf. autowiring configuration `excludes`), and only load this directory on services_test.yaml file

Reply:
Also, don't forget to remove the useless autoload of the `fixtures` directory in composer: https://github.com/crf-devs/resop/blob/master/composer.json#L70

Reply:
As the file is now in the src dir, adding the tag seems okay for me. And we could use it in dev fixtures also